scripts/provision-all

```diff
@@ -17,6 +17,11 @@ sudo qubesctl --show-output --skip-dom0 --targets sys-firewall state.sls sd-sys-
 echo "Set up dom0 config files, including RPC policies, and create VMs"
 sudo qubesctl --show-output state.highstate
 
+echo "Setup sd-log vm first"
+sudo qubesctl --show-output --skip-dom0 --targets sd-log state.sls sd-log-templates-files
+# Make sure that the VM is in shutdown state
+qvm-shutdown sd-log
```

Comment:
Do we still need `sd-log-reboot.sls` with this logic here? Surely `sd-log` should reboot if a log needs to be forwarded?